hello-world/host/src/test/java/com/r3/conclave/sample/host/NativeTest.java

```diff
@@ -0,0 +1,59 @@
+package com.r3.conclave.sample.host;
+
+import com.r3.conclave.host.AttestationParameters;
+import com.r3.conclave.host.EnclaveHost;
+import com.r3.conclave.host.EnclaveLoadException;
+import com.r3.conclave.host.MailCommand;
+import com.r3.conclave.mail.MailDecryptionException;
+import com.r3.conclave.mail.PostOffice;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledOnOs;
+import org.junit.jupiter.api.condition.OS;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import java.nio.file.Paths;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * This test assumes a Linux platform and will try to load the enclave for real, either in simulation mode, or
+ * when compiled with the correct -PenclaveMode= flag, in debug or release mode.
+ */
+@EnabledOnOs(OS.LINUX)
+public class NativeTest {
+    private static EnclaveHost enclave;
+    private static final List<MailCommand> mailCommands = new ArrayList<>();
+
+    @BeforeAll
+    static void startup() throws EnclaveLoadException {
+        enclave = EnclaveHost.load();
+        enclave.start(new AttestationParameters.DCAP(), null, Paths.get("./conclave.disk"), mailCommands::addAll);
```

Comment:
@shamsasari  is there any example that could help

Reply:
Yes, this test isn't testing that the enclave is able to read back previously persisted data. You need to restart the enclave to do this.

Since you're using the web host, your test should use that. Unfortunately I don't know off the top of my head how to do that. The web host is essentially a Spring boot web controller. Is there a way perhaps by calling the `EnclaveWebHost.main` method that you can spin up the web host and use it for testing.